src/ethereum/curve/contracts.ts

```diff
@@ -27,6 +30,9 @@ const contracts = {
   metaSwapAbi,
   zapAbi,
   metaZapAbi,
+  metapoolFactory,
```

Comment:
All Abis should follow this format: 

`CurveMetaPoolDeposit: { abi: CurveMetaPoolDepositAbi } ` 

If an address is needed:

`CurveMetaPoolDeposit: { 
        abi: CurveMetaPoolDepositAbi,
        address: "someAddress"
 } `

Reply:
With the correct indentation on the address field!